snippets/csharp/System.Globalization/DateTimeFormatInfo/FullDateTimePattern/dtfi_fulldatetimepattern.cs

```diff
@@ -26,8 +26,8 @@ public class SamplesDTFI  {
 This code produces the following output.  The question marks take the place of native script characters.
```

Comment:
Will be good to change the comment telling that this output is example which can be different depending on the used OS, the OS version, and the native globalization library used in such OS.

Reply:
Good point! Something like: `This code produces the following output. Note that the exact output format depends on the OS, the OS version, and the native globalization library used by the OS.` ?